src/hooks/useWalletSectionsData.ts

```diff
@@ -144,34 +136,48 @@ export default function useWalletSectionsData({
       sortedAssets,
       accountBalanceDisplay: accountWithBalance?.balancesMinusHiddenBalances,
       isLoadingBalance: !accountWithBalance?.balancesMinusHiddenBalances,
-      // @ts-expect-error ts-migrate(2698) FIXME: Spread types may only be created from object types... Remove this comment to see the full error message
-      ...isWalletEthZero,
+      isWalletEthZero,
       hiddenTokens,
       isReadOnlyWallet,
       listType: type,
       showcaseTokens,
-      uniqueTokens: allUniqueTokens,
+      uniqueTokens,
       isFetchingNfts,
-      nftSort,
       remoteConfig,
       experimentalConfig,
       positions,
       claimables,
-      claimableETHRewardsNativeAmount,
-    };
-
-    const { briefSectionsData, isEmpty } = buildBriefWalletSectionsSelector(accountInfo);
-    const hasNFTs = allUniqueTokens.length > 0;
-
-    return {
-      hasNFTs,
-      isEmpty,
-      isLoadingBalance: !accountWithBalance?.balances,
+      nftSort,
+      remoteCards,
+    }),
+    [
+      hiddenAssets,
+      isCoinListEdited,
       isLoadingUserAssets,
+      language,
+      nativeCurrency,
+      network,
+      pinnedCoins,
+      sendableUniqueTokens,
+      sortedAssets,
+      accountWithBalance?.balancesMinusHiddenBalances,
       isWalletEthZero,
-      briefSectionsData,
-    };
-  }, [
+      hiddenTokens,
+      isReadOnlyWallet,
+      type,
+      showcaseTokens,
+      uniqueTokens,
+      isFetchingNfts,
+      remoteConfig,
+      experimentalConfig,
+      positions,
+      claimables,
+      nftSort,
+      remoteCards,
+    ]
+  );
+
+  const { briefSectionsData, isEmpty } = useCachedSelector(buildBriefWalletSectionsSelector, walletSectionsState, [
```

Comment:
I think this should just be a single `useMemo` like it was before, these seem to share the same dependencies.